about-impl-days.md

```diff
@@ -0,0 +1,101 @@
+title: "What is `impl Days`"
+permalink: /about_impl_days/
+---
+
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; padding: 0; margin-bottom: 1em;">
+  <div class="popout">
+    <section>
+      <h1>impl Days</h1>
+    </section>
+  </div>
+</div>
+
+{% comment %}
+<section markdown="1">
+## Todays Info
+{% include_relative impl_days_room_info.html %}
+</section>
+{% endcomment %}
+
+<section markdown="1">
+## about:impl_days
+
+{% comment %}
+TODO: add sponsors
+{% endcomment %}
+
+`impl Days` is the sibling event of RustFest&lt;Rome&gt;. `impl Days` is a full-day event for Rust contributors to
+collaborate in an active programming session. It will take place on the two days following the RustFest talks and
+workshops, on {{site.dates.impl_days_no_br}} {{site.dates.year}}.
+
+## History
+
+Born as a hacking and development experiment in the early RustFest editions, the "impl Days" has established as a
+tradition following each RustFest.
+
+The name refers to the first edition that took place during the 2017 "impl Period", therefore the name "impl Days" was a
+natural choice.  <a href="https://internals.rust-lang.org/t/the-impl-period-newsletter-2/6034"> Over those two days</a>,
+30 to 40 people came together to discuss, design, and implement changes to Rust itself as well its broader ecosystem.
+
+Plus, <a href="https://botbot.me/mozilla/rust-internals/2017-10-02/?msg=91818378&page=2">they got to build a Lego Death
+Star</a>.
+
+The Zurich `impl Days` was such a success that we decided to repeat the event in Paris and it was an even <a
+href="https://twitter.com/MozillaParis/status/1001434063264272384" target="_blank">bigger success</a> (also, <a
+href="https://twitter.com/simukis/status/1001122476011802626" target="_blank">a rocket</a>).
+
+## `impl Days` Rome
+
+{% comment %}
+TODO: add info about {{site.location.impl_days}} and {{site.location.impl_days_overflow}}
+{% endcomment %}
+
+The event location(s) will be announced at a later stage.
+
+`impl Days` is a great chance to work directly with other Rust enthusiasts. The Rust project is supported by volunteers
+across the globe, and while we often rely on github comments and IRC channels for communication, the face-to-face
+communication enabled by an event like this can be a big productivity boost.
+
+You may be just starting out as a Rust contributor, or you may be a long-time compiler hacker who reminisces about the
+days when Rust was bootstrapped atop ML. Either way, there's plenty of tasks for all contributors at `impl Days`.
+
+You do not need to buy a separate ticket for `impl Days`; your Rust Fest ticket will suffice.
+
+Thus, everyone who plans to attend `impl Days` should tell us explicitly on the online questionnaire built into the
+RustFest ticket.
+
+## What will happen at the `impl Days`?
+
+{% comment %}
+TODO: add coordinator, schedule
+{% endcomment %}
+
+Schedule about the impl Days will announced when details will be finalized.
```

Comment:
The schedule for the impl Days will be announced closer to the event, when details are finalized